compiler/noirc_frontend/src/monomorphization/mod.rs

```diff
@@ -895,8 +896,8 @@ impl<'interner> Monomorphizer<'interner> {
             let mutable = false;
             let definition = Definition::Local(fresh_id);
             let name = i.to_string();
-            let typ = Self::convert_type(&field_type, location)?;
 
+            let typ = Self::convert_type(tuple_type, location)?;
```

Comment:
Could this be moved outside of the loop?

Reply:
Yes and no - we'd need another `Location` to pass since this one is on the tuple element